MyString.java

```diff
@@ -64,8 +98,12 @@ public static String spacedString(String str) {
      * @return a randomly generated string, consisting of 'n' lowercase letters
      */
     public static String randomStringOfLetters(int n) {
-        //// Replace the following statement with your code
-        return null;
+        String word = "";
+        for ( int i = 0; i < n; i++){
+          int letter =(int)((Math.random()) * 26) + 97;

```

Comment:
26, 97 magic numbers use array length or constant for the first, ASCII value for the second